shell/artifacts/hex/join-game.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2018 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+defineParticle(({SimpleParticle}) => {
+  return class extends SimpleParticle {
+    constructor() {
+      super();
+    }
+    getTemplate(slotName) {
+      return `<join-game on-click="onJoin">Join the game!</join-game>`;
```

Comment:
This is fine (and same in next particle) but there is some value to adding `html` destructured param at line 8 and setting to a const like:

````
const template = html`<div></div>`;
```

and then returning that here, in that editors end up doing html syntax highlighting, which is nice.